src/Sylius/Behat/Page/Admin/Product/UpdateConfigurableProductPage.php

```diff
@@ -50,6 +55,34 @@ public function isProductOptionsDisabled()
     /**
      * {@inheritdoc}
      */
+    public function isMainTaxonChosen($taxonName)
+    {
+        $this->openTaxonBookmarks();
+        
+        return $taxonName === $this->getDocument()->find('css', '.search > .text')->getText();
```

Comment:
What if `$this->getDocument()->find('css', '.search > .text')` will return `null`? We should add assertion for that.